desktop/apps/auction2/reducers.js

```diff
@@ -1,21 +1,123 @@
 import { combineReducers } from 'redux'
 import u from 'updeep'
+import { contains } from 'underscore'
+import { data as sd } from 'sharify'
+import * as actions from './actions'
 
 const initialState = {
+  minEstimateRangeDisplay: 50,
+  maxEstimateRangeDisplay: 50000,
   artworks: [],
-  isListView: false
+  total: 0,
+  isListView: false,
+  isFetchingArtworks: false,
+  sortMap: {
+    "lot_number": "Lot Number (asc.)",
+    "-lot_number": "Lot Number (desc.)",
+    "-searchable_estimate": "Most Expensive",
+    "searchable_estimate": "Least Expensive"
+  },
+  aggregatedArtists: [],
+  aggregatedMediums: [],
+  filterParams: {
+    size: 50,
+    page: 1,
+    sale_id: sd.AUCTION.id,
+    aggregations: ['TOTAL', 'MEDIUM', 'FOLLOWED_ARTISTS', 'ARTIST'],
+    sort: 'lot_number',
+    gene_ids: [],
+    artist_ids: [],
+    estimate_range: '',
+    ranges: {
+      estimate_range: {
+        min: 50,
+        max: 50000
+      }
+    }
+  }
 }
 
 function auctionArtworks(state = initialState, action) {
   switch (action.type) {
-  case 'UPDATE_ARTWORKS':
+  case actions.UPDATE_ARTWORKS:
     return u({
       artworks: action.payload.artworks
     }, state)
-  case 'TOGGLE_LIST_VIEW':
+  case actions.TOGGLE_LIST_VIEW:
     return u({
       isListView: action.payload.isListView
     }, state)
+  case actions.UPDATE_TOTAL:
+    return u({
+      total: action.payload.total
+    }, state)
+  case actions.UPDATE_SORT:
+    return u({
+      filterParams: {
+        sort: action.payload.sort
+      }
+    }, state)
+  case actions.UPDATE_AGGREGATED_ARTISTS:
+    return u({
+      aggregatedArtists: action.payload.aggregatedArtists
+    }, state)
+  case actions.UPDATE_AGGREGATED_MEDIUMS:
+    return u({
+      aggregatedMediums: action.payload.aggregatedMediums
+    }, state)
+  case actions.UPDATE_ESTIMATE_RANGE:
+    return u({
+      filterParams: {
+        estimate_range: `${action.payload.min * 100}-${action.payload.max * 100}`
+      }
+    }, state)
+  case actions.UPDATE_ESTIMATE_DISPLAY:
+    return u({
+      minEstimateRangeDisplay: action.payload.min,
+      maxEstimateRangeDisplay: action.payload.max
+    }, state)
+  case actions.UPDATE_ARTIST_ID:
+    const artistId = action.payload.artistId
+    if (artistId == 'artists-all') {
+      return u({
+        filterParams: {
+          artist_ids: []
+        }
+      }, state)
+    } else if (contains(state.filterParams.artist_ids, artistId)) {
+      return u({
+        filterParams: {
+          artist_ids: u.reject((aa) => { return aa == artistId })
+        }
+      }, state)
+    } else {
+      return u({
+        filterParams: {
+          artist_ids: state.filterParams.artist_ids.concat(artistId)
+        }
+      }, state)
+    }
+  case actions.UPDATE_MEDIUM_ID:
```

Comment:
No linter config. Sarah and I talked earlier about `standard`, but there's nothing to enforce any particular way just yet.